src/model/CarMover.js

```diff
@@ -0,0 +1,26 @@
+import { makeRandomNum } from '../utils/helperUtils.js'
+import { Car } from './Car.js'
+
+export class CarMover {
+  #raceCars
+
+  constructor(cars) {
+    this.#raceCars = this.#getRaceCars(cars)
+  }
+
+  #getRaceCars(cars) {
+    return cars.map((carName) => new Car(carName))
```

Comment:
```suggestion
    return cars.map(Car.of)
```

Car에 이런식으로 표현할 수 있도록 of라는 static method 를 하나 만들어주면 어떨까 싶네요!